### PR TITLE
OSDOCS-4480: corrects json syntax

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 1]
```diff
@@ -243,7 +243,7 @@ $ oc patch Network.config.openshift.io cluster \
 $ oc patch Network.config.openshift.io cluster \
   --type='merge' --patch '{
     "spec": {
-      "clusterNetwork": [
+      {"clusterNetwork": [
```
Comment (npinaeva) [inline modules/nw-ovn-kubernetes-migration.adoc:246]: I am not sure if this change fixes anything, the previous version actually looks correct if I count the braces properly?

Comment (JoeAldinger) [reply modules/nw-ovn-kubernetes-migration.adoc:246]: Yeah, 7 can't be right. Can we close this bug as `Not a bug` in your opinion, @npinaeva ?

Comment (npinaeva) [reply modules/nw-ovn-kubernetes-migration.adoc:246]: I would say you should ask bug reporter about that, maybe they meant something else

**Comment by JoeAldinger** [comment]
Not a bug

**Pull request closed without merging**